app/src/ingestion/edd/spiders/edd_spider.py

```diff
@@ -100,34 +102,46 @@ def to_markdown(self, html: str) -> str:
         )
         # Clean up markdown text: consolidate newlines; replace non-breaking spaces
         markdown = re.sub(r"\n\n+", "\n\n", markdown).replace("\u00A0", " ")
+
+        # Replace non-absolute URLs with absolute URLs. 2 scenarios:
+        parsed = urlparse(base_url)
+        domain_prefix = parsed.scheme + "://" + parsed.netloc + "/"
+        # Scenario 1: link starts with '/' like "/en/unemployment/"
+        # Prepend the domain prefix to the link
+        markdown = re.sub(r"\]\(\/", rf"]({domain_prefix}", markdown)
+        # Scenario 2: link does not start with '/' or "http://" or "https://", like "unemployment/"
+        # Insert the base URL of the web page before the link
```

Comment:
Could we extract this out to a function (`relative_to_absolute`) and add test coverage?

Reply:
How would you like the test to run, since it's in a separate project -- see https://github.com/navapbc/labs-decision-support-tool/pull/98#issuecomment-2415178235?

Reply:
I totally missed that comment, sorry! My initial thought was that this would be a useful utility function and would be something we import into this part of the codebase. Curious if that seems too much like mixing the two codebases to you though?

Reply:
Let me see if I can import it from our project without too much complexity.

Reply:
Did some `sys.path` tweaking in the EDD project.